src/cellranger/processes/count.nf

```diff
@@ -27,6 +27,24 @@ def generateCellRangerCountCommandDefaults = {
 	} else if (processParams.containsKey('chemistry')) {
 		_chemistry = processParams.chemistry
 	}
+
+	_includeIntrons = null
```

Comment:
I would advise to scope this variable with `def`. I've seen that it can lead to some weird behavior if this is note done.